src/infra/repos/postgresql/prisma/schema.prisma

```diff
@@ -31,18 +31,39 @@ model Pet {
   specieAlias String?  @map("specie_alias")
   petName     String   @map("pet_name")
   gender      String
+  breedAlias  String   @map("breed_alias")
```

Comment:
this need to be optional, like `specieAlias`, because this field only will be used when breed name is `other`.